.github/workflows/platform-zxc-release-maven-central.yaml

```diff
@@ -220,7 +220,7 @@ jobs:
         with:
           build-root-directory: platform-sdk
           gradle-version: ${{ inputs.gradle-version }}
-          arguments: release-maven-central --scan -PpublishSigningEnabled=true
+          arguments: releaseMavenCentral --scan -PpublishSigningEnabled=true
```

Comment:
❓ We do not have to change this, but using camelCase (and not kebab-case) is the _idiomatic_ way to name Gradle tasks

Reply:
This was a hold over from Maven profiles and was simply not a priority to change.